CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+# Copyright (c) 2019 ARM Limited
```

Comment:
It should probably be - 1st commit year -> -2019. Don't know what the 1st commit year is though.

Reply:
Repository 1st commit or file first commit?

Reply:
File 1st commit.

Reply:
According to [apache's website](https://apache.org/legal/src-headers.html) :

> The top of each NOTICE file should include the following text, suitably modified to reflect the product name and year(s) of distribution of the current and past versions of the product [...]

When only 1 year is displayed, I have also seen the first year of publication or the current year being used, I don't know what's the appropriate way to go TBH.

Reply:
Seems we're not following the rules otherwise, either -- Each source file should include the following license header -- note that there should be no copyright notice in the header:

Reply:
isn't the SPDX-License-Identifier tag some sort of notice ?